developer-docs-site/docs/tutorials/first-coin.md

```diff
@@ -10,6 +10,24 @@ import TabItem from '@theme/TabItem';
 
 This tutorial introduces how you can compile, deploy, and mint your own coin, named [MoonCoin](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/moon_coin).
 
+## Step 0: Install prerequisites
+
+If you don't already have them (confirmed with `which command-name`), install the following packages to take this tutorial:
+
+1. **Homebrew**: [https://brew.sh/](https://brew.sh/) by executing the following command in your terminal:
+    ````bash
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+    ````
+1. **pnpm**: Install the latest [pnpm](https://pnpm.io/):
+    ```bash
+    brew install pnpm
+    ```
```

Comment:
I'm very bothered by us assuming that the default dev is a mac user.

Reply:
> I'm very bothered by us assuming that the default dev is a mac user.

Brew works on Linux too. Greg and I have developed a survey to get hard data on environment.  Also, these packages are needed only for the tutorials as documented. Let's talk.

Reply:
> I wonder if we cannot make a md imported by another md, so we can create this once and share? Secondly, can we please, please either make this generic or make different tabs for Mac versus generic. I'm very uncomfortable with our preference toward Macs. Have we looked at our data to suggest that we can prioritize Mac developers?

@davidiw , as I understand these Prerequisites apply only to Mac, implying these are already on most Linux systems. We say nothing about Windows support.

For background, these Prerequisites were originally on the Getting Started page:
https://aptos.dev/guides/getting-started

But that seemed misplaced as they apply only to Mac and only to to the tutorials, it seems. So I moved them to the tutorials index:
https://aptos.dev/tutorials/aptos-quickstarts#install-macos-prerequisites

But then you requested they be removed from that page, so I filtered them into the respective tutorial.

So I no longer understand the desires here. Where should this information reside? What prerequisites do we know of for the other operating systems? Thanks.